concepts/segment.md

```diff
@@ -0,0 +1,18 @@
+---
+grand_parent: Resources
+parent: Concepts
+title: Segment
+description: A sequence of characters
+---
+
+A segment is a unit that is part of a document or other file that is processed by a Computer-Aided Translation tool (CAT).
```

Comment:
Since this article is about `segments`, then we should bold the first appearance of the term.

```
A **segment** is...
```

To avoid the repetition of "that is", we can reformulate.

```
A segment is a unit in a document or other file that is processed by a computer-aided translation (CAT) tool. 
```

Reply:
Done